reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -210,6 +210,21 @@ public final HttpServer forwarded(boolean forwardedEnabled) {
 		return dup;
 	}
 
+	/**
+	 * Configure the {@link HttpForwardedHeaderHandler}.
+	 *
+	 * @param handler the forwarded header handler
+	 * @return a new {@link HttpServer}
+	 * @since 0.9.12
+	 */
+	public final HttpServer forwardedHeaderHandler(HttpForwardedHeaderHandler handler) {
```

Comment:
let's name it `forwarded(HttpForwardedHeaderHandler handler)`
Is it possibile to use `BiFunction<ConnectionInfo, HttpRequest, ConnectionInfo> handler` instead of `HttpForwardedHeaderHandler`